script/dispatcher.lua

```diff
@@ -607,7 +606,16 @@ function ProcessRequest(reqIndex)
 
     if debug_log then log("  "..loadingListItem..", "..loadingList[i].count.." in "..loadingList[i].stacks.." stacks ") end
   end
-  global.Dispatcher.Deliveries[selectedTrain.id] = {force=requestForce, train=selectedTrain, started=game.tick, from=from, to=to, networkID=providerData.network_id, shipment=shipment}
+  global.Dispatcher.Deliveries[selectedTrain.id] = {

```

Comment:
to_id and from_id added to every delivery

Reply:
That is a problem I run into anyway when allowing to click station names. It does not matter if LTN provides the ID or LTNT tries to find it later. At the moment, with duplicate names being an error, that is imo acceptable.
But since I need a new system for 1.11.x anyway, I might as well revert this change and keep the current implementation for a while longer.